pkg/charts/dependencies.go

```diff
@@ -250,35 +250,28 @@ func UpdateHelmMetadataWithDependencies(fs billy.Filesystem, mainHelmChartPath s
 	if err != nil {
 		return err
 	}
-	// Pick up all existing dependencies tracked by Helm by name
-	helmDependencyMap := make(map[string]*helmChart.Dependency, len(dependencyMap))
-	for _, dependency := range chart.Metadata.Dependencies {
-		if _, ok := dependencyMap[dependency.Name]; !ok {
-			continue
-		}
-		helmDependencyMap[dependency.Name] = dependency
-	}
+
 	// Update the Repository for each dependency
 	for dependencyName := range dependencyMap {
-		d, ok := helmDependencyMap[dependencyName]
-		if !ok {
+		found := false
+		for _, d := range chart.Metadata.Dependencies {
+			if d.Name == dependencyName {
+				d.Repository = fmt.Sprintf("file://./charts/%s", dependencyName)
```

Comment:
Did we miss doing d.Version =. ""  ? here wantedly ?

Reply:
Good point - I had been looking at https://github.com/rancher/charts-build-scripts/issues/68 and was thinking that this remapping of repositories probably shouldn't mess with any existing version.

But that is an separate issue so I'll reinstate the version-removal logic here, and we can perhaps discuss the version issue separately via a follow-up PR.

Reply:
Okay thanks. Please update the PR and make a new PR

Reply:
Done!